spec/spec_helper.rb

```diff
@@ -1,2 +1,5 @@
 $LOAD_PATH.unshift File.expand_path(File.join(File.dirname(__FILE__), '../lib'))
 $LOAD_PATH.unshift File.expand_path(File.join(File.dirname(__FILE__), '../app'))
+
+require 'capybara/rspec'
+require 'dummy/config/environment'
```

Comment:
true, loading rails at this point doesn't make sense. I'll check which specs are not integration test but depend on rails (classes)